ansible/roles/screenly/files/screenly.conf

```diff
@@ -33,7 +33,7 @@ debug_logging = False
 ; Only display pages with proper SSL certificates
 verify_ssl = True
 
-; Run Resin wifi connect if there is no connection
+; Run Balena WiFi Connect if there is no connection
 enable_offline_mode = False
 
 ; Set to 'auth_basic' to use HTTP Basic Authentication (see below)
```

Comment:
Add a ticket for this to replace all auth code with Django based auth.

Reply:
Done. See #1918.